draft/spec/change-log.md

```diff
@@ -0,0 +1,108 @@
+"hand-drive"](https://avatars0.githubusercontent.com/u/35607965) by
+[Patrick Hochstenbach](http://orcid.org/0000-0001-8390-6171) is
+licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/).
+
+## Changes from OCFL v1.0 to v1.1
+
+[Version 1.1 of the OCFL Specification](https://ocfl.io/draft/spec/) is a [minor version](https://semver.org/) update to the [OCFL Specification v1.0](https://ocfl.io/1.0/spec/). It is intended to add functionality in a backwards compatible manner, and to clarify certain aspects based on a community feedback.
+
+### Additions in v1.1
+
+#### Add requirements version number sequence
+
+Added [Conformance of prior versions](https://ocfl.io/draft/spec/#conformance-of-prior-versions) section to clarify that existing version directories are immutable and that the version number sequence must be monaotonic. Adds error code [E103](https://ocfl.io/draft/spec/#E103). (Issue [#544](https://github.com/OCFL/spec/issues/544))
+
+### Clarifications in v1.1
+
+#### One conformance declaration per object and storage root
+
+Update the [Object Conformance Declaration](https://ocfl.io/draft/spec/#object-conformance-declaration) and [Root Conformance Declaration](https://ocfl.io/draft/spec/#root-conformance-declaration) sections to clarify that there must be exactly one version declaration file. Error codes [E003](https://ocfl.io/draft/spec/#003) and [E076](https://ocfl.io/draft/spec/#E076) correspondingly updated. (Issue [#581](https://github.com/OCFL/spec/issues/581))
+
+#### Inventory uses UTF-8 encoding
+
+In [Inventory](https://ocfl.io/draft/spec/#inventory) section, clarify that UTF-8 encoded JSON must be used for the `inventory.json` files. (Issue [#514](https://github.com/OCFL/spec/issues/514))
+
+#### Version naming convention
+
+Update wording in [Version Directories](https://ocfl.io/draft/spec/#version-directories) section to talk about version consistency for all versions of an object-at-rest, rather than in terms of the process for adding a version. (Issue [#541](https://github.com/OCFL/spec/issues/541))
+
+#### Clarify manifest block requirements
+
+Add language in [Manifest](https://ocfl.io/draft/spec/#manifest) section to clarify that the `manifest` block must be a JSON object (adding error code [E106](https://ocfl.io/draft/spec/#E106)) and that the each key must correspond to a digest value key found in one or more `state` blocks (adding error code [E107](https://ocfl.io/draft/spec/#E107)). (Issue [#537](https://github.com/OCFL/spec/issues/537))
+
+#### Clarify manifest requirements in historic inventories
+
+Wording of the [Content Directory](https://ocfl.io/draft/spec/#content-directory) section improved to make it clear that for each historical inventory, the manifest must reference every file in that version directory. (Issue [#538](https://github.com/OCFL/spec/issues/538))
+
+#### Clarify language and error codes for version numbers
+
```

Comment:
Suggestion from:
```
Change [Version Directories](https://ocfl.io/draft/spec/#version-directories) section changed to be more
```
to:
```
Change [Version Directories](https://ocfl.io/draft/spec/#version-directories) section to be more
```

Reply:
Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)